src/assets/css/_sass/elements/_navs.scss

```diff
@@ -12,28 +12,24 @@
   justify-content: space-between;
   padding: 30px 50px;
   width: 100%;
-}
-
-#main-nav {
-  display: flex;
-  flex-direction: row; // width: 400px;
-  justify-content: flex-start;
-  box-shadow: 3px 4px 5px 0px rgba(0, 0, 0, 0.75);
-  ul {
-    background-color: #2b2b2b;
-    display: flex;
-    flex-direction: row;
-    list-style: none;
-    margin-left: auto;
-    margin-bottom: 0;
-    li {
-      padding: 20px 35px 20px;
-      a {
-        color: $color-text--light;
-        font-family: Work Sans; // Temporary;
-      }
-      a:hover {
-        color: $color-grey--light;
+  #main-nav {
+    box-shadow: 3px 4px 5px 0px rgba(0, 0, 0, 0.75);
+    ul {
```

Comment:
Lists don't make the best navs. This case in particular it is more work than styling the links themselves inside the `nav` element